src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -224,35 +220,17 @@ private PermissionRequirements findPermissionRequirement(SearchTaskRequest searc
         }
     }
 
-    // TODO: Once the granular permission feature flag enabled or available_tasks_only parameter is depreciated,
-    // this method should only check AVAILABLE_TASK_ONLY context
     private boolean isAvailableTasksOnly(SearchTaskRequest searchTaskRequest) {
-        final EnumMap<SearchParameterKey, SearchParameterBoolean> boolKeyMap = asEnumMapForBoolean(searchTaskRequest);
-        SearchParameterBoolean availableTasksOnly = boolKeyMap.get(AVAILABLE_TASKS_ONLY);
 
         RequestContext context = searchTaskRequest.getRequestContext();
 
         if (context == null) {
-            return availableTasksOnly != null && availableTasksOnly.getValues();
+            return false;
         } else {
             return context.equals(RequestContext.AVAILABLE_TASKS);
         }
```

Comment:
you can simplify this like - return context != null && ontext.equals(RequestContext.AVAILABLE_TASKS);